Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM rabbitmq:3.8.2-alpine
+FROM bitnami/rabbitmq:3.8
```

Comment:
Please use full version to have reproducible builds.

Upgrading to latest version (sinc it is a minor upgrade) might make a sense .